buildSrc/src/main/kotlin/com/saveourtool/save/buildutils/DockerStackConfiguration.kt

```diff
@@ -138,24 +163,51 @@ fun Project.createStackDeployTask(profile: String) {
         commandLine("docker-compose", "--file", "$buildDir/docker-compose.yaml", "up", "-d", "mysql")
         errorOutput = ByteArrayOutputStream()
         doLast {
-            if (!errorOutput.toString().contains(" is up-to-date")) {
```

Comment:
because error output doesn't contain this string, it only report that container started, even in the logs there is no such string,
i tried to get log output but failed. seems like this is separate task